### ISSUE TITLE
Focus lock doesn\t work on Iphone with VoiceOver

### ISSUE THREAD

**Issue by DeanNeal** [theKashey/react-focus-lock#336]

**Comment by theKashey** [comment]
"Yes", locking focus is not enough.
You need or use https://github.com/theKashey/react-focus-on (scroll locking can be disabled) or at least accompany lock with https://github.com/theKashey/aria-hidden
